kotsadm/web/src/components/PreflightResultPage.jsx

```diff
@@ -247,14 +251,38 @@ class PreflightResultPage extends Component {
       this.setState({
         preflightResultData: response.preflightResult,
       });
-    } catch(err) {
+    } catch (err) {
       console.log(err);
       this.setState({
         errorMessage: err ? `Encountered an error while fetching preflight results: ${err.message}` : "Something went wrong, please try again."
       });
     }
   }
 
+  sendSkipPreflightsData = async () => {
+    const { slug } = this.props.match.params;
+
+    fetch(`${window.env.API_ENDPOINT}/app/${slug}/preflight/skip`, {
+      headers: {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+        "Authorization": Utilities.getToken(),
+      },
+      method: "POST",
+    })
+      .then((res) => {
+        if (res.status === 200) {
```

Comment:
`if` and `else` are identical here.  so is `catch`.
`sendSkipPreflightsData` doesn't need to wait for this call to complete before navigating to the app page.